projects/step-core/src/lib/modules/basics/services/generate-api-key.service.ts

```diff
@@ -11,10 +12,24 @@ export class GenerateApiKeyService implements GenerateApiKeyStrategy {
     this.strategy = strategy;
   }
 
-  showGenerateApiKeyDialog(): void {
+  showGenerateApiKeyDialog(): Observable<any> {
```

Comment:
Interface doesn't implemented properly. Service's method return types should be the same as in strategy